distributed/cli/tests/test_dask_remote.py

```diff
@@ -0,0 +1,29 @@
+from mock import Mock
+
+from click.testing import CliRunner
+
+from distributed.cli.dask_remote import remote, _remote
```

Comment:
I recommend pulling valuable functions out of the cli module.  We might want to keep that module as thin as possible.